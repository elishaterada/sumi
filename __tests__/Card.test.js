import React from 'react';
import renderer from 'react-test-renderer';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';

import {
  CardClass,
  CardImageClass,
  CardContentClass,
  CardContentItemClass,
} from '../src/components/Card';

expect.addSnapshotSerializer(createSerializer(emotion));

test('renders Card', () => {
  const reactComponent = renderer
    .create(
      <div
        className={CardClass}
      >
        <img
          className={CardImageClass}
          src="https://source.unsplash.com/ngcX09jzLE0/800x533"
          alt="Jelly Fish"
        />
        <div className={CardContentClass}>
          <p className={CardContentItemClass}>Movements living a fully ethical life silo emerging justice, benefit corporation do-gooder program areas changemaker inspire society black lives matter the resistance uplift.</p>
        </div>
      </div>,
    )
    .toJSON();

  expect(reactComponent).toMatchSnapshot();
});
