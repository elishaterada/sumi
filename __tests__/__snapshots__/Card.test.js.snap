// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card 1`] = `
<div
  className={undefined}
>
  <img
    alt="Jelly Fish"
    className={undefined}
    src="https://source.unsplash.com/ngcX09jzLE0/800x533"
  />
  <div
    className={undefined}
  >
    <p
      className={undefined}
    >
      Movements living a fully ethical life silo emerging justice, benefit corporation do-gooder program areas changemaker inspire society black lives matter the resistance uplift.
    </p>
  </div>
</div>
`;
